gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/PegasusPlugin.java

```diff
@@ -1783,28 +1783,24 @@ protected GenerateDataTemplateTask configureDataTemplateGeneration(Project proje
       dataTemplateJarDepends.add(prepareExtensionSchemasForPublishTask);
     }
 
+    // include pegasus files in the output of this SourceSet
+    project.getTasks().withType(ProcessResources.class).getByName(targetSourceSet.getProcessResourcesTaskName(), it ->
```

Comment:
LGTM but do we have any test assertions which verify the resources actually end up in the jar?

Reply:
There are, but in retrospect there are not enough and no targeted test for it. I went back and wrote a new integration test for this.